services/director-v2/tests/unit/test_modules_dask_client.py

```diff
@@ -471,7 +472,14 @@ def comp_run_metadata(faker: Faker) -> MetadataDict:
 
 @pytest.fixture
 def task_labels(comp_run_metadata: MetadataDict) -> ContainerLabelsDict:
-    return comp_run_metadata
+    return parse_obj_as(
+        ContainerLabelsDict,
+        {
+            k.replace("_", "-").lower(): v
```

Comment:
So, I think the comment is not needed because:
- *parse_obj_as* is a pydantic function that parses the value into a ContainerLabelsDict, which is a dict[DockerLabelKey, str]
- *DockerLabelKey* is a pydantic *ConstrainedStr* which is a str that uses a regex to impose the Docker label key rules (i.e. no *_* but only *-*)
Is this ok?